examples/examples/fetch_all_accounts.rs

```diff
@@ -38,9 +38,14 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let api = ClientBuilder::new()
         .build()
         .await?
-        .to_runtime_api::<polkadot::RuntimeApi<DefaultConfig, DefaultExtra<DefaultConfig>>>();
+        .to_runtime_api::<polkadot::RuntimeApi<DefaultConfig, DefaultExtra<DefaultConfig>>>()?;
```

Comment:
Is there currently any way to avoid doing the full metadata hash check each time?

It feels to me that the per-pallet check should be the default (as it is currently), but that the whole-metadata check should be opt-in rather than opt-out, because it doesn't really matter that the metadata differs as long as the pallets you interact with are ok (and potentially some computation can be saved since most people won't interact with all of the pallets anyway).

Reply:
Yep, that would simplify things quite a bit. Have changed the API to reflect this 😄 Thanks!